### PR TITLE
Use lazy class for delay initialization of services in localchannelContext

### PR DESCRIPTION
**Pull request by jatgarg**
Use built in support for delay initialization of services in local channel context.

### CODE REVIEW

File: examples/data-objects/client-ui-lib/package.json [hunk 1]
```diff
@@ -51,7 +51,7 @@
     "@fluid-example/fluid-object-interfaces": "^0.27.0",
     "@fluid-example/search-menu": "^0.27.0",
     "@fluid-internal/client-api": "^0.27.0",
-    "@fluidframework/common-utils": "^0.23.0",
+    "@fluidframework/common-utils": "^0.24.0-0",
```
Comment (anthony-murphy) [inline examples/data-objects/client-ui-lib/package.json:54]: would be good to do the bump in a seperate review

File: packages/runtime/datastore/src/localChannelContext.ts [hunk 2]
```diff
@@ -51,6 +52,7 @@ export class LocalChannelContext implements IChannelContext {
         dirtyFn: (address: string) => void,
         private readonly snapshotTree: ISnapshotTree | undefined,
     ) {
+        this.services = new Lazy(() => this.servicesGenerator());
```
Comment (anthony-murphy) [inline packages/runtime/datastore/src/localChannelContext.ts:55]: >servicesGenerator [](start = 44, length = 17)

i think i would prefer if this was just inclined, as the function should only be called once and here

**Review by anthony-murphy — CHANGES_REQUESTED** [review]
:clock1: misclick in code flow

**Review by anthony-murphy — APPROVED** [review]
:shipit:

File: examples/data-objects/client-ui-lib/package.json (hunk 1 above)
Comment (jatgarg) [reply examples/data-objects/client-ui-lib/package.json:54]: So lazy class is not exported in previous version of common-utils, so need to bump. Or are you saying bump first and then make modifications to localchannelcontext?

Comment (jatgarg) [reply examples/data-objects/client-ui-lib/package.json]: Done

File: packages/runtime/datastore/src/localChannelContext.ts (hunk 2 above)
Comment (jatgarg) [reply packages/runtime/datastore/src/localChannelContext.ts]: done

**Pull request merged**